docs/plugins/integrations/migrations.rst

```diff
@@ -0,0 +1,72 @@
+*************
+Plugin Schema
+*************
+
+.. contents:: Table of Contents
+
+The integration framework provides a means for plugins to better manage their schema. Queries are in migration files that match the plugin's versions number in it's config. When the a plugin is installed or upgraded, it will loop over the migration files up to the latest version.
```

Comment:
```suggestion
The Integration Framework provides a means for Plugins to better manage their schema. Queries are in migration files that match the Plugin's versions number in its config. When the a Plugin is installed or upgraded, it will loop over the migration files up to the latest version.
```